internal/pkg/template/templates/addons/aurora/env/rdws/serverlessv2.yml

```diff
@@ -0,0 +1,154 @@
+Parameters:
+  App:
+    Type: String
+    Description: Your application's name.
+  Env:
+    Type: String
+    Description: The environment name your service, job, or workflow is being deployed to.
+  # Customize your Aurora Serverless cluster by setting the default value of the following parameters.
+  {{logicalIDSafe .ClusterName}}DBName:
+    Type: String
+    Description: The name of the initial database to be created in the Aurora Serverless v2 cluster.
+    Default: {{.InitialDBName}}
+    # Cannot have special characters
+    # Naming constraints: https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/CHAP_Limits.html#RDS_Limits.Constraints
+  VPCID:
+    Type: String
+    Description: The ID of the VPC in which to create the Aurora Serverless v2 cluster.
+    Default: ""
+  PrivateSubnets:
```

Comment:
Do we need the accompanying `addons.parameters.yml` file or is that already added in a previous PR?

Reply:
Both `env/serverlessv2.yml` and `env/rdws/serverlessv2/yml` share the same `env/addons.parameters.yml` file:
https://github.com/aws/copilot-cli/blob/mainline/internal/pkg/template/templates/addons/aurora/env/addons.parameters.yml